### PR TITLE
Artifact LvL 1 HS Keyboard addition

### PR DESCRIPTION
**Pull request by yiancar**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/artifact/lvl/rev_hs01/info.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                { "matrix": [5, 2], "w": 1.5, "x": 2.5, "y": 5.25 },
+                { "matrix": [5, 6], "w": 7, "x": 4, "y": 5.25 },
+                { "matrix": [5, 11], "w": 1.5, "x": 11, "y": 5.25 },
+                { "matrix": [5, 12], "x": 12.5, "y": 5.25 },
+                { "matrix": [5, 13], "w": 1.5, "x": 13.5, "y": 5.25 },
+                { "matrix": [5, 14], "x": 15.25, "y": 5.25 },
+                { "matrix": [5, 15], "x": 16.25, "y": 5.25 },
+                { "matrix": [5, 16], "x": 17.25, "y": 5.25 }
+            ]
+        }
```
Comment (dunk2k) [inline keyboards/artifact/lvl/rev_hs01/info.json:144]:
```suggestion
        },
        "LAYOUT_tkl_ansi_wkl": {
            "layout": [
                { "matrix": [0, 0], "x": 0, "y": 0 },
                { "matrix": [0, 1], "x": 2, "y": 0 },
                { "matrix": [0, 2], "x": 3, "y": 0 },
                { "matrix": [0, 3], "x": 4, "y": 0 },
                { "matrix": [0, 4], "x": 5, "y": 0 },
                { "matrix": [0, 5], "x": 6.5, "y": 0 },
                { "matrix": [0, 6], "x": 7.5, "y": 0 },
                { "matrix": [0, 7], "x": 8.5, "y": 0 },
                { "matrix": [0, 8], "x": 9.5, "y": 0 },
                { "matrix": [0, 9], "x": 11, "y": 0 },
                { "matrix": [0, 10], "x": 12, "y": 0 },
                { "matrix": [0, 11], "x": 13, "y": 0 },
                { "matrix": [0, 12], "x": 14, "y": 0 },
                { "matrix": [0, 14], "x": 15.25, "y": 0 },
                { "matrix": [0, 15], "x": 16.25, "y": 0 },
                { "matrix": [0, 16], "x": 17.25, "y": 0 },
                { "matrix": [1, 0], "x": 0, "y": 1.25 },
                { "matrix": [1, 1], "x": 1, "y": 1.25 },
                { "matrix": [1, 2], "x": 2, "y": 1.25 },
                { "matrix": [1, 3], "x": 3, "y": 1.25 },
                { "matrix": [1, 4], "x": 4, "y": 1.25 },
                { "matrix": [1, 5], "x": 5, "y": 1.25 },
                { "matrix": [1, 6], "x": 6, "y": 1.25 },
                { "matrix": [1, 7], "x": 7, "y": 1.25 },
                { "matrix": [1, 8], "x": 8, "y": 1.25 },
                { "matrix": [1, 9], "x": 9, "y": 1.25 },
                { "matrix": [1, 10], "x": 10, "y": 1.25 },
                { "matrix": [1, 11], "x": 11, "y": 1.25 },
                { "matrix": [1, 12], "x": 12, "y": 1.25 },
                { "matrix": [1, 13], "w": 2, "x": 13, "y": 1.25 },
                { "matrix": [1, 14], "x": 15.25, "y": 1.25 },
                { "matrix": [1, 15], "x": 16.25, "y": 1.25 },
                { "matrix": [1, 16], "x": 17.25, "y": 1.25 },
                { "matrix": [2, 0], "w": 1.5, "x": 0, "y": 2.25 },
                { "matrix": [2, 1], "x": 1.5, "y": 2.25 },
                { "matrix": [2, 2], "x": 2.5, "y": 2.25 },
                { "matrix": [2, 3], "x": 3.5, "y": 2.25 },
                { "matrix": [2, 4], "x": 4.5, "y": 2.25 },
                { "matrix": [2, 5], "x": 5.5, "y": 2.25 },
                { "matrix": [2, 6], "x": 6.5, "y": 2.25 },
                { "matrix": [2, 7], "x": 7.5, "y": 2.25 },
                { "matrix": [2, 8], "x": 8.5, "y": 2.25 },
                { "matrix": [2, 9], "x": 9.5, "y": 2.25 },
                { "matrix": [2, 10], "x": 10.5, "y": 2.25 },
                { "matrix": [2, 11], "x": 11.5, "y": 2.25 },
                { "matrix": [2, 12], "x": 12.5, "y": 2.25 },
                { "matrix": [3, 12], "w": 1.5, "x": 13.5, "y": 2.25 },
                { "matrix": [2, 14], "x": 15.25, "y": 2.25 },
                { "matrix": [2, 15], "x": 16.25, "y": 2.25 },
                { "matrix": [2, 16], "x": 17.25, "y": 2.25 },
                { "matrix": [3, 0], "w": 1.75, "x": 0, "y": 3.25 },
                { "matrix": [3, 1], "x": 1.75, "y": 3.25 },
                { "matrix": [3, 2], "x": 2.75, "y": 3.25 },
                { "matrix": [3, 3], "x": 3.75, "y": 3.25 },
                { "matrix": [3, 4], "x": 4.75, "y": 3.25 },
                { "matrix": [3, 5], "x": 5.75, "y": 3.25 },
                { "matrix": [3, 6], "x": 6.75, "y": 3.25 },
                { "matrix": [3, 7], "x": 7.75, "y": 3.25 },
                { "matrix": [3, 8], "x": 8.75, "y": 3.25 },
                { "matrix": [3, 9], "x": 9.75, "y": 3.25 },
                { "matrix": [3, 10], "x": 10.75, "y": 3.25 },
                { "matrix": [3, 11], "x": 11.75, "y": 3.25 },
                { "matrix": [3, 13], "w": 2.25, "x": 12.75, "y": 3.25 },
                { "matrix": [4, 0], "w": 2.25, "x": 0, "y": 4.25 },
                { "matrix": [4, 2], "x": 2.25, "y": 4.25 },
                { "matrix": [4, 3], "x": 3.25, "y": 4.25 },
                { "matrix": [4, 4], "x": 4.25, "y": 4.25 },
                { "matrix": [4, 5], "x": 5.25, "y": 4.25 },
                { "matrix": [4, 6], "x": 6.25, "y": 4.25 },
                { "matrix": [4, 7], "x": 7.25, "y": 4.25 },
                { "matrix": [4, 8], "x": 8.25, "y": 4.25 },
                { "matrix": [4, 9], "x": 9.25, "y": 4.25 },
                { "matrix": [4, 10], "x": 10.25, "y": 4.25 },
                { "matrix": [4, 11], "x": 11.25, "y": 4.25 },
                { "matrix": [4, 12], "w": 2.75, "x": 12.25, "y": 4.25 },
                { "matrix": [4, 15], "x": 16.25, "y": 4.25 },
                { "matrix": [5, 0], "w": 1.5, "x": 0, "y": 5.25 },
                { "matrix": [5, 2], "w": 1.5, "x": 2.5, "y": 5.25 },
                { "matrix": [5, 6], "w": 7, "x": 4, "y": 5.25 },
                { "matrix": [5, 11], "w": 1.5, "x": 11, "y": 5.25 },
                { "matrix": [5, 13], "w": 1.5, "x": 13.5, "y": 5.25 },
                { "matrix": [5, 14], "x": 15.25, "y": 5.25 },
                { "matrix": [5, 15], "x": 16.25, "y": 5.25 },
                { "matrix": [5, 16], "x": 17.25, "y": 5.25 }
            ]
        }
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/artifact/lvl/rev_hs01/halconf.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* Copyright 2021 Harrison Chan (Xelus)
```
Comment (keyboard-magpie) [inline keyboards/artifact/lvl/rev_hs01/halconf.h:1]: add yiancar

File: keyboards/artifact/lvl/rev_hs01/mcuconf.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/* Copyright 2021 Harrison Chan (Xelus)
+ *
```
Comment (keyboard-magpie) [inline keyboards/artifact/lvl/rev_hs01/mcuconf.h:2]: add yiancar

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]
minor changes in copyright headers

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/artifact/lvl/rev_hs01/info.json
Comment (fauxpark) [inline keyboards/artifact/lvl/rev_hs01/info.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/artifact/lvl/rev_hs01/config.h [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* RGB Light */
+#define WS2812_PWM_DRIVER PWMD1
+#define WS2812_PWM_CHANNEL 2
+#define WS2812_PWM_PAL_MODE 2
```
Comment (fauxpark) [inline keyboards/artifact/lvl/rev_hs01/config.h:23]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/artifact/lvl/rev_hs01/config.h (hunk 4 above)
Comment (yiancar) [reply keyboards/artifact/lvl/rev_hs01/config.h:23]: Is this now set automatically?

Comment (yiancar) [reply keyboards/artifact/lvl/rev_hs01/config.h:23]: Ah its the defaults

**Review by fauxpark — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**